tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/QuietState.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <mutex>
+
+#include <QuietBackgroundProcesses.h>
+#include "QuietState.h"
+
+namespace QuietState
+{
+    std::mutex g_mutex;
+
+    void TurnOff()
+    {
+        auto lock = std::scoped_lock(g_mutex);
+        try

```

Comment:
>try [](http://example.com/codeflow?start=8&length=3)

Minor: DisableQuietBackgroundProcesses returns an HRESULT, so it shouldn't be throwing anything.